### DISCUSSION TITLE
Can we get the sats off of legend DEAD SERVERs?

### DISCUSSION THREAD

**Discussion by midizzz** [lnbits/lnbits#2532] [Q&A]
feels like I missed the memo. small amount of sats but never like to burn them. 
DEAD SERVER, ONLY FOR EXPORTING TRANSACTION DATA AND LNURLAUTH

**Comment by bitkarrot** [comment]
No. And you were never supposed to use legend for anything other than super short term testing.

Run your own instance. That's the lesson here.

**Reply by midizzz** [reply]
Thanks for the fast response
